browser/services/data.js

```diff
@@ -15,6 +13,7 @@ class InstallerDataService {
     this.downloading = false;
     this.installing = false;
 
+    let env = require('remote').require('../main/env');
```

Comment:
Is this move purely to make the declaration of 'env' closer to its use or another reason?